src/program/OutfitProject.cpp

```diff
@@ -1135,19 +1135,19 @@ void OutfitProject::SetTextures(NiShape* shape, const std::vector<std::string>&
 
 			if (!mat.Failed()) {
 				if (mat.signature == MaterialFile::BGSM) {
-					texFiles[0] = mat.diffuseTexture.c_str();
```

Comment:
The c_str() is needed to skip duplicate NULL terminators that were added to the strings from IO. Without the c_str(), the strings will end up at a size of 1 with \0 as the explicit content, which will make my !texFiles[i].empty() check a few lines below fail.

Until I find a way to fix the \0 terminators in IO after reading the material files, this has change has to be removed (same in PreviewWindow.cpp). I might need to resize the string to c_str() after reading it from the file.